app/src/main/kotlin/com/aliucord/manager/ui/screen/InstallScreen.kt

```diff
@@ -88,36 +89,40 @@ fun InstallerScreen(
                     name = "Download APKs",
                     isCurrent = viewModel.currentCategory == InstallViewModel.StepCategory.APK_DL,
                     subSteps = listOf(
-                        viewModel.baseApkDl,
-                        viewModel.librariesApkDl,
-                        viewModel.localeApkDl,
-                        viewModel.resourceApkDl
+                        viewModel.steps[InstallViewModel.Steps.DL_BASE_APK]!!,
+                        viewModel.steps[InstallViewModel.Steps.DL_LIBS_APK]!!,
```

Comment:
Given that the step will always be in the map perhaps something like InstallViewModel#getStep -> !null